pom.xml

```diff
@@ -37,7 +37,7 @@
     <url>https://github.com/CAFDataProcessing/worker-workflow</url>
 
     <properties>
-        <caf.worker-document-framework.version>4.2.0-SNAPSHOT</caf.worker-document-framework.version>
+        <caf.worker-document-framework.version>4.2.0-SCMOD-7711-SNAPSHOT</caf.worker-document-framework.version>
```

Comment:
Revert this change now that CAFDataProcessing/worker-document#40 has been merged.